tests/integration/test_gcloud/test_configuration/test_remote_configuration.py

```diff
@@ -94,13 +94,9 @@ def get_remote_configuration(component_name, config):
     try:
         if rec_msg_ok.startswith('ok'):
             remote_configuration = json.loads(rec_msg)
-            if host_type == 'server':
```

Comment:
If this check is no longer necessary, shouldn't we remove the assignment in line 79?

Reply:
+1

Reply:
Fixed!